notebook/static/notebook/js/codecell.js

```diff
@@ -313,7 +313,7 @@ define([
             stop_on_error = true;
         }
 
-        this.output_area.clear_output(false, true);
+        this.clear_output(false, true);
```

Comment:
@jasongrout I think that `this.clear_output` should be used instead of `this.output_area.clear_output` because `this.clear_output` triggers events whereas `this.output_area.clear_output` doesn't.